app/views/layouts/application.html.erb

```diff
@@ -18,7 +18,7 @@
     <ul id="user-login-controls" class="pull-right">
       <% if user_signed_in? %>
         <li><small>Signed in as: </small><strong><%= current_user.email %></strong></li>
-        <li><%= link_to "Edit Profile", edit_user_registration_path, class: "navbar-link" %></li>
+        <li><%= link_to "Profile", user_path(current_user.id), class: "navbar-link" %></li>
```

Comment:
`user_path(...)` method can just be given a `User` object, its optional to call `.id` on it:

```diff
- user_path(current_user.id)
+ user_path(current_user)
```